libraries/AP_Scripting/applets/BatteryTag.md

```diff
@@ -0,0 +1,19 @@
+# Battery Tag Support
+
+This script implements logging of DroneCAN BatteryTag messages. It is
+used in combination with a BatteryTag AP_Periph node to log
+information about number of cycles a battery has been through along
+with the serial number and number of hours in an armed state.
+
+The log message in BTAG
```

Comment:
Did you forget to finish a thought?

Reply:
Pretty sure this needs to be filled out with some